component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/Constants.java

```diff
@@ -47,5 +47,29 @@ public static class SQLQueries {
         public static final String GET_JWT = "SELECT EXP_TIME,TIME_CREATED FROM IDN_OIDC_JTI WHERE JWT_ID =?";
         public static final String INSERT_JWD_ID = "INSERT INTO IDN_OIDC_JTI (JWT_ID, EXP_TIME, TIME_CREATED)" +
                 "VALUES (?,?,?)";
+        public static final String INSERT_OR_UPDATE_JWT_ID_MSSQL_OR_DB2 = "MERGE IDN_OIDC_JTI T USING  " +
```

Comment:
Are these queries tested with different types of DBs like oracle, SQL ?

Reply:
Yes, description is updated and please find the tested DBs